doc/cookbook/input_processor.rst

```diff
@@ -0,0 +1,139 @@
+.. index::
+   single: Input; Custom input processor
+
+How to Create a custom Input processor
+======================================
+
+RollerworksSearch already provides input processors for a wide range of
+formats, including XML, JSON, PHP Array's and the user-friendly FilterQuery.
```

Comment:
https://github.com/sstok/RollerworksSearch/blob/doc-update-and-fixes/doc/input/filter_query.rst ;) it's a special format for text input.